src/apps/Settings/DaoSettings.js

```diff
@@ -122,27 +129,39 @@ class DaoSettings extends React.Component {
             </Note>
           </Option>
         )}
-        {apmApps.length > 0 && (
+        {appsLoading && (
+          <Option name="Aragon apps" text={'Loading apps…'}>
+            <div css={'height:20px'} />
+          </Option>
+        )}
+        {!appsLoading && apmApps.length > 0 && (
           <Option
             name="Aragon apps"
             text={`This organization has ${apmApps.length}
             ${apmApps.length > 1 ? 'apps' : 'app'}
             installed.`}
           >
             <AppsList>
-              {apmApps.map(({ appId, description, name, proxyAddress }) => {
-                const checksummedProxyAddress = toChecksumAddress(proxyAddress)
-                return (
-                  <li title={description} key={checksummedProxyAddress}>
-                    <Field label={name}>
+              {apmApps.map(
+                ({ appId, description, name, proxyAddress, tags }) => {
+                  const checksummedProxyAddress = toChecksumAddress(
+                    proxyAddress
+                  )
+
+                  return (
+                    <li title={description} key={checksummedProxyAddress}>
+                      <Label>
+                        {name}
+                        {tags.length > 0 ? ` (${tags.join(', ')})` : ''}
```

Comment:
You can use the same default value tool here: `{ ..., tags =[] }` then `tags.join(', ')` will work (empty array will give you empty string).

Reply:
We'd still need the ternary as it not only adds the enumeration but also adds the parenthesis to the copy

Reply:
True, sorry I missed the parenthesis